event/network.go

```diff
@@ -0,0 +1,16 @@
+package event
+
+import (
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+// EvtPeerConnectednessChange should be emitted every time we form a connection with a peer or drop our last
+// connection with the peer. Essentially, it is emitted in two cases:
+// a) We form a/any connection with a peer.
+// b) We go from having a connection/s with a peer to having no connection with the peer.
+// It contains the Id of the remote peer and the new connectedness state.
+type EvtPeerConnectednessChange struct {
+	RemotePeerId  peer.ID
```

Comment:
nit: To We already know it's a peer event, not a local event, by its name. Also, we know it's an ID from the type. So in line with all other events, we should rename this field to `Peer`.

Reply:
And actually, I just realised another self-nit: it should be `EvtPeerConnectednessChanged` (past).

Reply:
See https://github.com/libp2p/go-libp2p-core/blob/master/event/doc.go#L8.